# ROS message containing NMPC state measurements
# Thomas Stastny | thomas.stastny@mavt.ethz.ch

float32 pos_n       # northing position [m]
float32 pos_e       # easting position [m]
float32 pos_d       # down position [m]
float32 airsp       # airspeed [m/s]
float32 fpa         # flight path angle [rad]
float32 heading     # heading angle [rad]
float32 roll        # roll angle [rad]
float32 pitch       # pitch angle [rad]
float32 prop_speed  # propeller speed [rps]
